src/client/src/components/pages/participants/participantCourseDetails/index.jsx

```diff
@@ -2,31 +2,61 @@ import React, { Component } from "react";
 import { state as initialState, fields as fieldSet } from "./staticData";
 import Form from "./../../../abstract/Form";
 import Footer from "../../../abstract/footer";
+import axios from 'axios';
+import contextHoc from './../../../abstract/HOC/contextHoc';
 
-export default class index extends Component {
+class index extends Component {
   state = initialState;
 
   goBack = event => {
-    this.props.history.push("/participants/courses");
+    const { id } = this.props.match.params;
+    this.props.history.push(`/participants/${id}/courses`);
   };
 
   // Fake Function because onChange is required in Form
   onChange = () => {
     return null;
   }
```

Comment:
we need to make changes on the form ( to make  `onChange( )` `null` by default instead of doing it here) to get cleaner code, we can make this edit in a new pull request.